requirements-wally.txt

```diff
@@ -1,4 +1 @@
-https://github.com/ElementsProject/libwally-core/releases/download/release_0.8.0/wallycore-0.8.0-cp36-cp36m-linux_x86_64.whl; 'linux' in sys_platform and python_version == '3.6' --hash=sha256:95809c3ef620317a9e3826600189f164dd69abe9c61b52a35e655e22396cc56f
-https://github.com/ElementsProject/libwally-core/releases/download/release_0.8.0/wallycore-0.8.0-cp37-cp37m-linux_x86_64.whl; 'linux' in sys_platform and python_version == '3.7' --hash=sha256:f287bb7c15155dac6ba866744fccdf3855e648c1b4366cbd67a75a7e353f567f
-https://github.com/ElementsProject/libwally-core/releases/download/release_0.8.0/wallycore-0.8.0-cp37-cp37m-macosx_10_14_x86_64.whl; 'darwin' in sys_platform and python_version == '3.7' --hash=sha256:6687bd503908f5f4a7052f622b03881c5217ae9c2810f5d28284eb22cdcbde22
-https://github.com/ElementsProject/libwally-core/archive/release_0.8.0.tar.gz --hash=sha256:a52b9cd46a7acc5031d8f1deb49bd51525966e38c5dc5544412e841593608ba6
+wallycore==0.8.1 --hash=sha256:ce8a62ad607a321740de0faf0cf34481c1932aa4c11168ad96b07789d0aaede3
```

Comment:
I'd add all hashes:
```
wallycore==0.8.1 \
    --hash=sha256:0ea264d6852c9bd6d73ec4603116e2fd7b52f287ae96dcc3adfd4f761b76ec59 \
    --hash=sha256:42b286a046ac2af9c98f7e5a9c6c2f34ee61c9969c45a99f1f3865dadd7d9eec \
    --hash=sha256:523b5f8f161428dce0643bb5a2efa9fab2f33accfd3be9276d48421e46686822 \
    --hash=sha256:55c3fc79182eae1eda068ce9ef4d199c22407ec2ef0d0b278af8f6770bf4a49e \
    --hash=sha256:6aa57488c80c2a40f00d96040fb6ec8c42bebc60a9363fb536788b593fafc024 \
    --hash=sha256:6cb711aa8ece8d1bf1f8bf2e8d9e00bff84550ee030e6ea0809480114b85cffe \
    --hash=sha256:6e2bdc5cf77479eb9d070f10b3e313540c30775b6704218b05626abdd7585b84 \
    --hash=sha256:6f83a421b703330398de1207f836302b5495f5be71bf335f5be93ba4e61c8a7a \
    --hash=sha256:73b7bba9e95c098437428d797a1ea3bbc4f83b0acdc43562e23323fb4eb43837 \
    --hash=sha256:848ea57236582daac5400131f155b0724ed24d6d1f60eb4d56802492e7c83c6a \
    --hash=sha256:90f65430e2c90c5c7f6993b5077e6de9485b3b57615524022ba319387f92ec02 \
    --hash=sha256:a9998e4ebadc87c646b4722961539cae615f4b4acc227ea98db6e1700494e332 \
    --hash=sha256:a9c0a0a4b13f94fbd9a7beac43ce3cd56cfcce2f5ce42bb69f6106f65ce27449 \
    --hash=sha256:acf04910cbe70af1e9eed6f7418bdfee35071a3148cac382d0f1eafe96a28edf \
    --hash=sha256:c5f228f0ba0460443cb6682a0e03b7240d89893bd2a218742c16e9ecccdfe2f0 \
    --hash=sha256:ce8a62ad607a321740de0faf0cf34481c1932aa4c11168ad96b07789d0aaede3 \
    --hash=sha256:e7428c21519bfa8a563ef4713d17ef050d70a2c9cdc6bd0d161301cb436c464b \
    --hash=sha256:f1a9bacf4049fcc84b57a8e2ae4d8f197f9185a7663bf3c43509574551bc3399 \
    --hash=sha256:f6716c50d33b3a39594c6abaf6e7aade9dbf901ce2e345790be5312a5297bd0f \
    --hash=sha256:f7fb7d64a1b78526d05cb872af0b36a391b58685d2ff9f70711cf106dca55cf9 \
    --hash=sha256:fd7b03e8d9c00bacd1832dff3be35912d191be5fe93230ef567ab7d517d1e2d1
```

(got this from an internal source and double checked against https://pypi.org/project/wallycore/#files)

Reply:
added, thanks